frontend/src/components/pages/CampOverview/MoveCampersModal.tsx

```diff
@@ -0,0 +1,256 @@
+            >
+              <Checkbox
+                className="camper-checkbox"
+                isChecked={camperIsSelected.every(Boolean)}
+                onChange={() => {
+                  if (!camperIsSelected.every(Boolean)) {
+                    setCamperIsSelected(
+                      Array<boolean>(campers.length).fill(true),
+                    );
+                  } else {
+                    setCamperIsSelected(
+                      Array<boolean>(campers.length).fill(false),
+                    );
+                  }
+                }}
+              >
+                Select all
+              </Checkbox>
+              {campers.map((campersItem, i) => (
+                <Checkbox
+                  className="camper-checkbox"
+                  key={i}
+                  isChecked={camperIsSelected[i]}
+                  onChange={() => {
+                    setCamperIsSelected(
+                      camperIsSelected.map((camperItem, camperIdx) => {
+                        if (camperIdx === i) {
+                          return !camperItem;
+                        }
+                        return camperItem;
+                      }),
+                    );
+                  }}
+                >{`${campersItem.firstName} ${campersItem.lastName}`}</Checkbox>
+              ))}
+            </Stack>
+          </ModalBody>
+
+          <ModalFooter>
+            <Button onClick={moveCamperModalOnClose}>Cancel</Button>
+            <Button
+              variant="solid"
+              style={{
+                backgroundColor: "#468740",
+                color: "white",
+                marginLeft: "10px",
+              }}
+              onClick={() => {
+                setModalPage(2);
+              }}
+              disabled={
+                camperIsSelected.reduce<number>(
+                  (prev, cur) => (cur ? prev + 1 : prev),
+                  0,
+                ) === 0
+              }
+            >
+              Next
+            </Button>
+          </ModalFooter>
+        </>
+      );
+    }
+    if (modalPage === 2) {
+      return (
+        <>
+          <ModalHeader>
+            Move{" "}
+            {camperIsSelected.reduce<number>(
+              (prev, cur) => (cur ? prev + 1 : prev),
+              0,
+            )}{" "}
+            Camper(s)
+          </ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <p style={{ marginBottom: "20px" }}>
+              You can move the camper(s) to another session in this camp by
+              selecting it below:
+            </p>
+            {campSessions.length > 0 ? (
+              <Select
+                value={selectedCampSession}
+                onChange={(e) => setSelectedCampSession(e.target.value)}
+              >
+                {campSessions.map((campSessionsItem, campSessionsIdx) => (
+                  <option key={campSessionsIdx} value={campSessionsItem.id}>
```

Comment:
we need to "disable" the option if the camp session is the current session or a session in the past. We also need to disable the session if it doesn't have enough spots available. 
<img width="601" alt="image" src="https://user-images.githubusercontent.com/57159970/198167258-3c22448b-b320-43d3-99fb-79b2522477d1.png">